### PR TITLE
Update changelog for v0.7.0

### LINKED ISSUE #580: Releasing from main as compatible with http4s 1.0.0-M31
Hi,

The current (March 1st 2022) main branch remains unreleased so we have no versions compatible with http4s 1.0.0-M31.

Can the project owners consider releasing from main a new milestone please?

The particular interest we have is support for proxies in the client. To c;confirm, our internal code works well with a custom local version built directly from main.

Cheers

### PR DESCRIPTION
**Pull request by amesgen**
Closes #580

### CODE REVIEW

**Review by rossabaker — COMMENTED** [review]
Does this changelog format work with Laika?  I've had to massage everything on http4s core because of duplicate headers.  It's about the most manual part of our release process left.

**Comment by armanbilge** [comment]
I think it was a bug that is now fixed in 0.18.1.

> Link validation now allows duplicate headers on the same page as long as no internal link points to them

https://github.com/planet42/Laika/releases

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,22 @@ This file summarizes **notable** changes for each release, but does not describe
...
+
+### Breaking changes
+
+* We now use the upstream WebSocket model made public in http4s-0.23.11.
+
+### Dependency updates
+
+* scala-2.12.15, scala-2.13.8, scala-3.1.1
+* cats-2.7.0
+* cats-effect-3.2.5
```
Comment (armanbilge) [inline CHANGELOG.md:19]: Latest CE is 3.3.8.
https://github.com/typelevel/cats-effect/releases/tag/v3.3.8

Comment (amesgen) [reply CHANGELOG.md:19]: Ah yes, that was fixed in #596

**Comment by rossabaker** [comment]
> Link validation now allows duplicate headers on the same page as long as no internal link points to them

Yay.  Translating this is the worst remaining part of doing releases on the core repo!

**Pull request merged**